content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -588,6 +588,10 @@ SPECIAL_TYPE
 If there are some files in the `/etc/config/` directory, they will be deleted.
 {{< /caution >}}
 
+{{< note >}}
+Text data is exposed as files using the UTF-8 character encoding. To use some other character encoding, use binaryData.
+{{< /note >}}
+
```

Comment:
@fml2 I think it's important to add a note like this into https://kubernetes.io/docs/concepts/configuration/configmap/

either as well as here, or instead of it. Do you agree?

Reply:
I'm undecided on this. On the one hand you are right. On the other hand, this is not the main place where volume usage is documented, but rather an example of configMap usage. Hence it does not need to describe all corner cases.

Besides, this would result in the note being present _three_ times (where twice is already not very good :-) So I'd rather not add it at this place.